bookkeeper-common/src/main/java/org/apache/bookkeeper/common/util/MathUtils.java

```diff
@@ -40,21 +40,6 @@ public static int findNextPositivePowerOfTwo(final int value) {
         return 1 << (32 - Integer.numberOfLeadingZeros(value - 1));
     }
 
-    /**
-     * Current time from some arbitrary time base in the past, counting in
-     * milliseconds, and not affected by settimeofday or similar system clock
-     * changes. This is appropriate to use when computing how much longer to
-     * wait for an interval to expire.
-     *
-     * <p>NOTE: only use it for measuring.
-     * http://docs.oracle.com/javase/1.5.0/docs/api/java/lang/System.html#nanoTime%28%29
-     *
-     * @return current time in milliseconds.
-     */
-    public static long now() {
-        return System.nanoTime() / NANOSECONDS_PER_MILLISECOND;
```

Comment:
Can't we call here System.currentTimeMillis and leave the rest unchanged?

Reply:
That is confusing. I would rather use System.currentTimeMillis to be more explicit